youtube/services.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from .models import Video
+from DjangoTut.settings import YOUTUBE_API_ACCESS_KEY
+
+
+class YoutubeService:
+    API_KEY = None
+    METHOD_GET = 'get'
+    METHOD_POST = 'post'
+    METHOD_DELETE = 'delete'
+
+    def __init__(self):
+        self.BASE_API_URL = "https://www.googleapis.com/youtube/v3/"
+
+        self.api_key = YOUTUBE_API_ACCESS_KEY
+        if self.api_key is None:
```

Comment:
można po prostu napisać `if not self.api_key`, wtedy jak pusty string będzie, to też wywali